python/src/FORM_doc.i.in

```diff
@@ -3,6 +3,10 @@
 
 Refer to :ref:`form_approximation`.
 
+Available constructors:
+    FORM(*nearestPointAlgorithm, event, physicalStartingPoint*)
```

Comment:
we dont need a list of available ctors when there is only one available: the list of parameters is enough